README.md

```diff
@@ -1,31 +1,695 @@
-# Module2 Project
+# Module-2 Project
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Implement a class with instance variables, a constructor, and methods that
+  access and mutate the instance variables.
+- Implement an application with multiple classes.
+- Implement various types of class associations.
 
-## Introduction
+## Instructions
 
-A 1-2 sentence summary of what will be covered.
+Fork and clone this project.
+The starter code contains several empty classes that
+you will develop incrementally as you learn the material
+in this module.
 
-## Instructions
+![uml module2 project](https://curriculum-content.s3.amazonaws.com/6676/java-multipleclasses/uml_module2_project.png)
+
+## Task #1 - The `Concert` Class
+
+The `Concert` class encapsulates data about:
+- the artist performing at a music concert
+- the number of tickets available for purchase
+- the number of people on the wait list
+
+(1) Add instance variables in the following order to the `Concert` class:
+
+- A String named `performer`.
+- An int named `available`.
+- An int named `waitlist`.
+
+(2) Add a constructor that takes two parameters to initialize the `performer` and
+`available` instance variables.  The `waitlist` instance variable
+should be initialized to a default value of 0.
+
+(3) Use IntelliJ to generate getter methods for each of the instance variables.
+
+(4) Use IntelliJ to generate a `toString()` method, selecting all fields.
+
+(5) Edit the `ConcertTest` Junit test class to test the constructor, getters,
+and `toString` methods:
+
+```java
+class ConcertTest {
+
+  private Concert c1, c2;
+
+  @BeforeEach
+  void setup() {
+    c1 = new Concert("The Weeknd", 10);
```

Comment:
Oh when I forked the module 2 project to create the module 3 project, I realized this was a typo in a lot of the unit tests.

"Weeknd" should be "Weekend"

Reply:
Looks like that is their name, "The Weeknd".